kernel-modules/build/utils.sh

```diff
@@ -37,6 +37,24 @@ setup_driver_config() {
 
     # Create the driver_config.h file
     envsubst < "${falco_dir}/driver/driver_config.h.in" > "${falco_dir}/driver/driver_config.h"
+
+    if [[ -d "${falco_dir}/driver/bpf/configure" ]]; then
+        for dir in "${falco_dir}"/driver/bpf/configure/*; do
+            if [[ ! -d "${dir}" ]]; then
+                continue
+            fi
+
+            CONFIGURE_MODULE=$(basename "$dir")
+            CONFIGURE_ROOT="${falco_dir}/driver/bpf/${CONFIGURE_MODULE}"
+
+            mkdir -p "${CONFIGURE_ROOT}"
+            cp "${dir}/test.c" "${CONFIGURE_ROOT}"
```

Comment:
The process involves creating a series of directories with `test.c` combined with a bunch of common makefiles and build scripts, to create generic configuration checks in the build. We _could_ do this in place, but in a local build this will be writing to git-tracked files so it would be a bit messy with source control

Reply:
Got it, thanks. But where do they land exactly, does `${falco_dir}` in this context points to something inside cmake build directory?

Reply:
no, `falco_dir` is in the source tree, `<collector_root>/falcosecurity-libs/`. This part of the drivers build process is just to configure the sources and then package them up and depending on the version we may or may not use cmake at all, so it felt cleaner and simpler to copy to a neutral, untracked location